draft-specifications/CapTP Specification.md

```diff
@@ -646,6 +646,129 @@ resolver when a resolution is available to the `listen-desc` object. If a
 resolution is already available, the resolver provided in `listen-desc` MUST be
 fulfilled or broken.
 
+## `op:get`
+
+`op:get` requests the value for the named field of an eventually settled
+[Struct](Model.md#Struct).  The get operation follows promise resolutions,
+inheriting the rejection reason of any intermediate rejected promise.
+The operation rejects the answer if the ultimate fulfillment of the receiver
```

Comment:
I think we should also specify that `op:get`s where the struct does not have the given `field-name` causes a rejection too.